__tests__/app.test.js

```diff
@@ -26,28 +26,26 @@ describe("GET/api/topics", () => {
       .get("/api/tpics")
       .expect(404)
       .then(({ body }) => {
-        expect.objectContaining({});
+        expect(body).toEqual({});
       });
   });
 });
 
-// describe("GET/api/articles/:article_id", () => {
-//   test("200: return article by id", () => {
-//     return request(app)
-//       .get("/api/articles/1")
-//       .expect(200)
-//       .then(({ body }) => {
-//         expect(body).toEqual([
-//           {
-//             article_id: 1,
-//             title: "Living in the shadow of a great man",
-//             topic: "mitch",
-//             author: "butter_bridge",
-//             body: "I find this existence challenging",
-//             created_at: "2020-07-09T20:11:00.000Z",
-//             votes: 100,
-//           },
-//         ]);
-//       });
-//   });
-// });
+describe("GET/api/articles/:article_id", () => {
+  test("200: return article by id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect.objectContaining(body);
+      });
+  });
+  test("404:article id not in database", () => {
```

Comment:
The test should be more specific and check whether you are receiving the actual error message you are sending.